script.tablo/addon.xml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.tablo"
+       name="Tablo"
+       version="1.0.0"
+       provider-name="Nuvyyo">
+    <requires>
+        <import addon="xbmc.python" version="2.14.0"/>
```

Comment:
That would leave it unusable on previous versions. Since I can't put it in the older repos, I was hoping to keep the version as is. Is this a hard and fast rule? I thought this was only needed if you used features that were only available on that version.